SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1067,6 +1070,12 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 
 			Worker_OpList_Destroy(OpList);
 		}
+
+		if (SpatialMetrics != nullptr)
```

Comment:
I think being able to turn metrics on/off should be configurable in the SpatialOS settings. @m-samiec @improbable-valentyn what do you think?